src/Codeception/Lib/Connector/Laminas.php

```diff
@@ -36,25 +56,22 @@ public function setApplicationConfig(array $applicationConfig)
 
     /**
      * @param BrowserKitRequest $request
-     *
-     * @return Response
-     *
      * @throws Exception
      */
-    public function doRequest($request)
+    public function doRequest($request): Response
     {
         $this->createApplication();
 
         $laminasRequest = $this->application->getRequest();
-        $uri            = new HttpUri($request->getUri());
+        $uri            = new Http($request->getUri());
```

Comment:
Http is a horrible and misleading class name here, could you leave the alias?

Reply:
Done.